apps/test/unit/templates/plugins/detailsTest.js

```diff
@@ -0,0 +1,237 @@
+  });
+
+  it('can render markdown syntax in the summary', () => {
+    const markdown =
+      '::: details [**summary** _content_]\n' +
+      'contents, which are sometimes further block elements\n' +
+      ':::';
+    const expected =
+      '<details><summary><strong>summary</strong> <em>content</em></summary><p>contents, which are sometimes further block elements</p></details>\n';
+
+    const rendered = parser.sourceToHtml(markdown);
+    assert.equal(rendered, expected);
+  });
+
+  it('can render markdown syntax in the body', () => {
+    const markdown =
+      '::: details [summary-content]\n' +
+      '\n' +
+      '# Contents\n' +
+      '- can\n' +
+      '- be\n' +
+      '- markdown\n' +
+      ':::';
+    const expected =
+      '<details><summary>summary-content</summary><h1>Contents</h1><ul>\n' +
+      '<li>can</li>\n' +
+      '<li>be</li>\n' +
+      '<li>markdown</li>\n' +
+      '</ul></details>\n';
+
+    const rendered = parser.sourceToHtml(markdown);
+    assert.equal(rendered, expected);
+  });
+
+  it('ignores trailing colons', () => {
+    // Look how pretty this can be!
+    const markdown =
+      '::::::::::::: details [summary-content] :::::::::::::\n' +
+      'contents, which are sometimes further block elements\n' +
+      ':::::::::::::::::::::::::::::::::::::::::::::::::::::';
+    const expected =
+      '<details><summary>summary-content</summary><p>contents, which are sometimes further block elements</p></details>\n';
+
+    const rendered = parser.sourceToHtml(markdown);
+    assert.equal(rendered, expected);
+  });
+
+  it('ignores excess whitespace', () => {
+    const markdown =
+      ':::      details       [summary-content]          \n' +
+      '\n' +
+      'contents, which are sometimes further block elements\n' +
+      '\n' +
+      ':::';
+    const expected =
+      '<details><summary>summary-content</summary><p>contents, which are sometimes further block elements</p></details>\n';
+
+    const rendered = parser.sourceToHtml(markdown);
+    assert.equal(rendered, expected);
+  });
+
+  it('can nest', () => {
+    const markdown =
+      ':::: details [outer]\n' +
+      '::: details [inner]\n' +
+      'innermost content\n' +
+      ':::\n' +
+      '::::';
+    const expected =
+      '<details><summary>outer</summary><details><summary>inner</summary><p>innermost content</p></details></details>\n';
+
+    const rendered = parser.sourceToHtml(markdown);
+    assert.equal(rendered, expected);
+  });
+
+  it('requires a summary block', () => {
+    const markdown =
+      '::: details\n' +
+      'contents, which are sometimes further block elements\n' +
+      ':::';
+    const expected =
+      '<p>::: details\n' +
+      'contents, which are sometimes further block elements\n' +
+      ':::</p>\n';
+
+    const rendered = parser.sourceToHtml(markdown);
+    assert.equal(rendered, expected);
+  });
+
+  it('requires at least three opening colons', () => {
```

Comment:
Is this requirement mentioned anywhere?

Reply:
This is also mentioned in the original proposal